### PR TITLE
Handle Country Selector

### LINKED ISSUE #1761: PanEuropean: country selector -> Handle Forest Europe instead of Global
in country selector, handle Forest Europe instead of Global

Image

### LINKED ISSUE #1765: PanEuropean landing: do not use dashboard data
accessing http://localhost:9000/FE/assessments/pan_european/2020/home/overview gives error because it's using useDashboardData.ts

Image

### PR DESCRIPTION
**Pull request by persas**
Closes #1761 
Closes #1765 

Note: I pushed to this branch the second issue by mistake, but I guess it is OK.

### CODE REVIEW

File: src/client/hooks/useIsPath.ts [hunk 1]
```diff
@@ -20,3 +21,10 @@ export const useIsPrint = () => ({
   print: useIsPath({ path: `${ClientRoutes.Assessment.Print.path.absolute}/*`, exact: false }),
   onlyTables: useIsPath({ path: `${ClientRoutes.Assessment.PrintTables.path.absolute}`, exact: true }),
 })
+
+export const useIsPanEuropean = () => {
+  const { assessmentName } = useParams<{ assessmentName: AssessmentName }>()
+  return assessmentName === 'paneuropean' || assessmentName === 'panEuropean'
+}
+
+// Add isPanEuropean - Check Assessment - useParams
```
Comment (sorja) [inline src/client/hooks/useIsPath.ts]: Extra comment can be removed

**Review by sorja — APPROVED** [review]

**Comment by minotogna** [comment]
screenshot or video please

**Comment by persas** [comment]
NOTE: For this screenshot the init is hardcoded to panEuropean
image

File: src/client/pages/AssessmentHome/PanEuropeanHome/PanEuropeanHome.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation()
+  const dispatch = useAppDispatch()
+
+  useEffect(() => {
+    dispatch(AssessmentActions.initApp({ assessmentName: 'panEuropean' }))
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```
Comment (minotogna) [inline src/client/pages/AssessmentHome/PanEuropeanHome/PanEuropeanHome.tsx]: are we sure this is needed here ? shouldn't this work without this?

File: src/client/hooks/useIsPath.ts [hunk 3]
```diff
@@ -20,3 +21,8 @@ export const useIsPrint = () => ({
   print: useIsPath({ path: `${ClientRoutes.Assessment.Print.path.absolute}/*`, exact: false }),
   onlyTables: useIsPath({ path: `${ClientRoutes.Assessment.PrintTables.path.absolute}`, exact: true }),
 })
+
+export const useIsPanEuropean = () => {
+  const { assessmentName } = useParams<{ assessmentName: AssessmentName }>()
+  return assessmentName === 'paneuropean' || assessmentName === 'panEuropean'
```
Comment (minotogna) [inline src/client/hooks/useIsPath.ts]: here better to use AssessmentNames.panEuropean instead of 'panEuropean'

Comment (sorja) [reply src/client/hooks/useIsPath.ts]: assessmentName should never be fully lowercase

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**